sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserService.java

```diff
@@ -290,8 +290,12 @@ public List<UserReference> getUserReferences(
 			userEntityJoinUsed = true;
 		}
 		if (filterByJurisdiction) {
-			filter = CriteriaBuilderHelper.and(cb, filter, createCurrentUserJurisdictionFilter(cb, userRoot));
-			userEntityJoinUsed = true;
+			if (hasRight(UserRight.SEE_PERSONAL_DATA_OUTSIDE_JURISDICTION)) {
```

Comment:
Actually, the conjunction stuff is probably pretty unnecessary; you can shorten this section to

```
if (!hasRight(...)) {
  filter = CriteriaBuilderHelper.and(cb, filter, createCurrentUserJurisdictionFilter(....));
  ...
}
```